src/opera/pge/dist_s1/dist_s1_pge.py

```diff
@@ -693,8 +694,6 @@ def _flatten_output_dir(self):
                 dst = os.path.join(output_product_path, basename(filename))
 
                 if scratch_path not in src and src != dst:
-                    # TODO: Change this to shutil.move() with the next SAS delivery. We need the output directory
-                    #  structure intact for the comparison script since it works on the whole directory
```

Comment:
Just so we have it documented somewhere, why are we removing this TODO without acting on it?